exporter/observiqexporter/converter.go

```diff
@@ -126,7 +126,7 @@ func resourceAndInstrumentationLogToEntry(resMap map[string]interface{}, log plo
 
 func timestampFromRecord(log plog.LogRecord) string {
 	if log.Timestamp() == 0 {
-		return timeNow().UTC().Format(timestampFieldOutputLayout)
+		return log.ObservedTimestamp().AsTime().UTC().Format(timestampFieldOutputLayout)
```

Comment:
Is `ObservedTimestamp` guaranteed to be non-zero?

Reply:
According to the specification, yes. But it's probably wise to check here in case there's a mistake upstream.

Reply:
Yeah, that's right. I added check for zero observed timestamp to not break things